lang/main.json

```diff
@@ -300,13 +301,24 @@
         "sessTerminated": "Call terminated",
         "sessionRestarted": "Call restarted by the bridge",
         "Share": "Share",
+        "shareAudio": "Continue",
+        "shareAudioTitle" : "How to share audio",
+        "shareAudioWarningTitle": "You need to stop screen sharing before sharing audio",
+        "shareAudioWarningH1": "If you want to share just audio:",
+        "shareAudioWarningD1": "you need to stop screen sharing before sharing your audio.",
+        "shareAudioWarningD2": "you need to restart your screen sharing and check the \"share audio\" option.",
+        "shareMediaWarningGenericH2": "If you want to share your screen and audio",
         "shareVideoLinkError": "Please provide a correct youtube link.",
         "shareVideoTitle": "Share video",
         "shareYourScreen": "Share your screen",
         "shareYourScreenDisabled": "Screen sharing disabled.",
         "startLiveStreaming": "Start live stream",
         "startRecording": "Start recording",
         "startRemoteControlErrorMessage": "An error occurred while trying to start the remote control session!",
+        "shareScreenWarningTitle": "You need to stop audio sharing before sharing your screen",
+        "shareScreenWarningH1": "If you want so share just your screen:",
```

Comment:
i feel that this should be either a 'to' instead of 'so' or a comma aftewards

Reply:
yup typo, fixed.